packages/frontity-org-theme/src/components/styles/global-styles.tsx

```diff
@@ -3,12 +3,24 @@ import FrontityOrg from "../../../types";
 import { addAlpha } from "../../utils";
 
 const cssReset = css`
-  html,
-  body {
-    border: none;
-    margin: 0;
-    padding: 0;
+  *,
+  *::before,
+  *::after {
+    box-sizing: border-box;
   }
+
+  html {
+    line-height: 1.15;
+    -webkit-text-size-adjust: 100%;
```

Comment:
why do we need that? :)

Reply:
The property prevents adjustments of font size after a change in orientation in iOS.

Reply:
Doesn't the "auto vendor-prefixing" feature of Emotion means that we don't need to use prefixes like `-webkit`?

Reply:
Yeah! I didn't know about that. I'm gonna fix it.